sysutils/just/Portfile

```diff
@@ -2,28 +2,94 @@
 
 PortSystem          1.0
 PortGroup           github 1.0
+PortGroup           cargo 1.0
 
-github.setup        casey just 0.2.0
+github.setup        casey just v0.4.4
```

Comment:
We don't want the `v` in the version. To fix that, use this instead:

`github.setup        casey just 0.4.4 v
`

Reply:
Fixed! Thanks.